src/view/pages/Brands/Profile/ProfileScreen.js

```diff
@@ -0,0 +1,193 @@
+import React, { useState } from "react";
+import styles from "./ProfileScreen.module.scss";
+import classNames from "classnames/bind";
+import Button from "../../../../components/Button/Button";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEye, faEyeSlash } from "@fortawesome/free-solid-svg-icons";
+
+const cx = classNames.bind(styles);
+
+export default function ProfileScreen() {
+  //   const changeImage = (e) => {
+  //     var preImage = document.getElementById("preview-img");
+  //     preImage.src = URL.createObjectURL(e.target.files[0]);
+  //     preImage.onload = () => {
+  //       URL.revokeObjectURL(output.src);
+  //     };
+  //   };
+  const [eye, setEye] = useState("none");
+  const [showPassword, setShowPassword] = useState("none");
+  const handleEyeClose = () => {
+    setEye("none");
+    setShowPassword(!showPassword);
+  };
+  const handleEyeOpen = () => {
+    setEye("block");
+    setShowPassword(!showPassword);
+  };
+  return (
+    <section className={cx("profile", "animation")}>
+      <div className={cx("container")}>
+        <div className={cx("profile-container")}>
+          <p className={cx("profile-path", "heading-small")}>
+            <span>Home</span>
+            <span>Profile</span>
+          </p>
+          <h2 className={cx("profile-title", "heading")}>
+            Brand’s information

```

Comment:
Research about multiple-languages using `react-i18next` lib, so we can also use English as well as Vietnamese!